src/main/resources/liquibase/202310041633-fix-placeholder-values-for-rosa.xml

```diff
@@ -0,0 +1,23 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="202310041633-1" author="ksynvrit">
+        <comment>Change four_vcpu_hour to Cores on subscription_measurements.</comment>
+        <update tableName="subscription_measurements">
+            <column name="metric_id" value="Cores"/>
+            <where>metric_id='four_vcpu_hour'</where>
+        </update>
+    </changeSet>
+
+    <changeSet id="202310041633-2" author="ksynvrit">
+        <comment>Change control_plane to Instance-hours on subscription_measurements.</comment>
+        <update tableName="subscription_measurements">
+            <column name="metric_id" value="Instance-hours"/>
```

Comment:
It appears that the `subscription_measurements` table currently has `INSTANCE_HOURS` in capital letters and with underscore. Additionally, you should update the Enum to string conversion as we spoke, or you may create a card and add that as a comment in the capacity call method.